test/web-platform-tests/to-upstream/html/interaction/focus-managment/focused-area-removal.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Focused area removal</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
```

Comment:
```suggestion
<body>
<script>
```

I tried running this test in the browser and it failed because of this. This isn't caught by jsdom's CI because our script loading is not fully synchronous :(.

Reply:
I originally included the `body` in a related PR (https://github.com/jsdom/jsdom/pull/3005) but you removed it [`1c3641a` (#3005)](https://github.com/jsdom/jsdom/pull/3005/commits/1c3641a24c32887f08a3e5db4d57e9cf61d54719) :wink:

Reply:
Oh, dang, my bad! That will cause us problems when upstreaming; I guess I'll have to fix it then.